python_modules/dagster/dagster_tests/execution_tests/dynamic_tests/test_dynamic_execution.py

```diff
@@ -12,84 +12,77 @@
     op,
     reconstructable,
 )
+from dagster._core.definitions.decorators.op_decorator import do_not_attach_code_origin
 from dagster._core.errors import DagsterExecutionStepNotFoundError
 from dagster._core.execution.api import create_execution_plan
 from dagster._core.execution.plan.state import KnownExecutionState
 from dagster._core.test_utils import instance_for_test
 from dagster._utils.merger import merge_dicts
 
+with do_not_attach_code_origin():
```

Comment:
per offline convo, let's move this to a pytest.fixture so it's easier to use in test and change less test code.